### PR TITLE
UI: part 3 remove alert banner

### PR DESCRIPTION
**Pull request by hellobontempo**
This PR has replaces _all_ uses of `<AlertBanner>` and some follow-up design items plus removing the old AlertBanner component

 Things to lookout for:
- [ ] If no custom title, does the Alert have the correct default (`Info`, `Warning` or `Error`)
- [ ] _Most_ `<Hds::Alert>` components should have `class="has-bottom-margin-s" ` added as a class
- [ ] Are there copy/pasta typos

### CODE REVIEW

File: ui/app/templates/components/clients/dashboard.hbs [hunk 1]
```diff
@@ -45,13 +45,17 @@
...
   {{else}}
     {{#if (eq @model.config.enabled "Off")}}
-      <AlertBanner data-test-tracking-disabled @type="warning" @title="Tracking is disabled">
-        Tracking is currently disabled and data is not being collected. Historical data can be searched, but you will need to
-        <LinkTo @route="vault.cluster.clients.edit">
-          edit the configuration
-        </LinkTo>
-        to enable tracking again.
-      </AlertBanner>
+      <Hds::Alert @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
```
Comment (hellobontempo) [inline ui/app/templates/components/clients/dashboard.hbs:48]: Screenshot 2023-06-16 at 4 33 12 PM

File: ui/app/templates/components/clients/dashboard.hbs [hunk 2]
```diff
@@ -93,24 +97,27 @@
...
+          <A.Description>
+            <ul class={{if (and this.versionUpdateText this.startTimeDiscrepancy) "bullet"}}>
+              {{#if this.startTimeDiscrepancy}}
+                <li>{{this.startTimeDiscrepancy}}</li>
+              {{/if}}
+              {{#if this.upgradeDuringActivity}}
+                <li>
+                  {{this.upgradeVersionAndDate}}
+                  {{this.upgradeExplanation}}
+                  <DocLink
```
Comment (hellobontempo) [inline ui/app/templates/components/clients/dashboard.hbs:111]: Plan is to keep `<DocLink>` but use the HDS component in that template

File: ui/lib/core/addon/components/message-error.js
Comment (hellobontempo) [inline ui/lib/core/addon/components/message-error.js:1]: There is a tech debt ticket to address the `<MessageError>` component and how/if we want to continue using it

File: ui/lib/pki/addon/components/page/pki-configure-create.hbs
Comment (hellobontempo) [inline ui/lib/pki/addon/components/page/pki-configure-create.hbs:1]: Design decided `@color="highlight"` is more appropriate for `Next steps` banners
Screenshot 2023-06-16 at 4 57 31 PM

File: ui/lib/pki/addon/components/page/pki-tidy-status.hbs [hunk 3]
```diff
@@ -28,11 +28,11 @@
     @color={{this.tidyStateAlertBanner.color}}
     @icon={{this.tidyStateAlertBanner.icon}}
     class="has-top-margin-m"
-    data-test-hds-alert
+    data-test-tidy-status-alert
```
Comment (hellobontempo) [inline ui/lib/pki/addon/components/page/pki-tidy-status.hbs:31]: trying to make selectors more component-relevant so debugging is easier in the future

**Review by zofskeez — APPROVED** [review]
🚀 🚢

**Pull request merged**